### PR TITLE
fix header

### PR DESCRIPTION
**Pull request by Yujinius0**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -21,23 +21,18 @@ a {
...
+.header {
     background-color: #f7033c;
-    width: 100%;
-    display: flex;
     height: 80px;
 }
 
-.header {
-    background-color: #f7033c;
+.container {
```
Comment (dtsymbalenko) [inline css/style.css]: Change to .header__container

File: css/style.css [hunk 2]
```diff
@@ -62,9 +57,21 @@ ul {
 
 .list__link {
     position: relative;
+    text-transform: uppercase;
+}
+
+.list__link-active {
```
Comment (dtsymbalenko) [inline css/style.css]: Change class name according to BEM convention

File: index.html [hunk 3]
```diff
@@ -65,32 +66,44 @@
...
+                            Акцент на качестве результата
+                        </p>
+                    </td>
+                    <td class="forma__title">
+                        <p class="forma__title-cell">
+                            Проекты любой сложности
+                        </p>
+                    </td>
+                </tr>
+            </table>
```
Comment (dtsymbalenko) [inline index.html]: What are non-header related changes doing here and below?

File: index.html [hunk 4]
```diff
@@ -100,7 +113,7 @@
                 <a class="inline" href="#">© 2018 Все права защищены. </a>
                 <ul class="list">
                     <li class="list__item">
-                        <a href="#" class="list__link">КОМПАНИЯ</a>
+                        <a href="#" class="list__link::before">КОМПАНИЯ</a>
```
Comment (dtsymbalenko) [inline index.html]: Where did you see such class writing?

File: css/style.css [hunk 5]
```diff
@@ -229,27 +236,35 @@ ul {
...
 }
 
+.forma__title-cell {
+    margin: 0;
+}
+
+.forma__title-cell3{
+    margin-top: 16px;
+}
+
```
Comment (dtsymbalenko) [inline css/style.css]: What are non-header related changes doing here? Incorrect class naming .forma__title-cell3

**Review by dtsymbalenko — CHANGES_REQUESTED** [review]

File: css/style.css (hunk 1 above)
Comment (Yujinius0) [reply css/style.css]: Name changed

File: css/style.css (hunk 2 above)
Comment (Yujinius0) [reply css/style.css]: Class name changed

File: css/style.css (hunk 5 above)
Comment (Yujinius0) [reply css/style.css]: Changes back to originals

File: index.html (hunk 3 above)
Comment (Yujinius0) [reply index.html]: Changes back to originals

File: index.html (hunk 4 above)
Comment (Yujinius0) [reply index.html]: Class name corrected

File: css/style.css [hunk 6]
```diff
@@ -60,7 +60,7 @@ ul {
     text-transform: uppercase;
 }
 
-.list__link-active {
+.list__link_active {
```
Comment (dtsymbalenko) [inline css/style.css]: Use '--' for modificators. .list__link--active

**Review by dtsymbalenko — CHANGES_REQUESTED** [review]

**Review by dtsymbalenko — APPROVED** [review]

**Pull request merged**